ibm-mq-mixin/panels.libsonnet

```diff
@@ -0,0 +1,589 @@
+local g = import './g.libsonnet';
+local commonlib = import 'common-lib/common/main.libsonnet';
+
+{
+  new(this)::
+    {
+      local signals = this.signals,
+
+      /*
+      -------------------------
+      Cluster Overview
+      -------------------------
+      */
+
+      clusterClusters:
+        commonlib.panels.generic.stat.info.new('Clusters', targets=[signals.cluster.clusters.asTarget()])
+        + g.panel.stat.panelOptions.withDescription('The unique number of clusters being reported.')
+        + g.panel.stat.standardOptions.withUnit('none'),
+
+      clusterQueueManagers:
+        commonlib.panels.generic.stat.info.new('Queue managers', targets=[signals.cluster.queueManagers.asTarget()])
+        + g.panel.stat.panelOptions.withDescription('The unique number of queue managers in the cluster being reported.')
+        + g.panel.stat.standardOptions.withUnit('none'),
+
+      clusterTopics:
+        commonlib.panels.generic.stat.info.new('Topics', targets=[signals.cluster.topics.asTarget()])
+        + g.panel.stat.panelOptions.withDescription('The unique number of topics in the cluster.')
+        + g.panel.stat.standardOptions.withUnit('none'),
+
+      clusterQueues:
+        commonlib.panels.generic.stat.info.new('Queues', targets=[signals.cluster.queues.asTarget()])
+        + g.panel.stat.panelOptions.withDescription('The unique number of queues in the cluster.')
+        + g.panel.stat.standardOptions.withUnit('none'),
+
+      clusterQueueOperations:
```

Comment:
I wonder if it'd make sense to show the labels + values outright on the pie chart?

Not sure myself, would you find that useful as an end user? @aalhour what do you think?